test/fixtures/flags-tasks-with-nowrap.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var gulp = require('gulp');
+
+gulp.task('task1', function() {});
+gulp.task('task1').description = 'Description for gulp.task("task1")';
+
+gulp.task('task2', function() {});
+if (!gulp.task('task2').unwrap) {
```

Comment:
The purpose of this test is that `gulp-cli` supports both the old way, `description` is gotten by `gulp.task(name)`, and the new way, it is gotten by `gulp.task(name).unwrap()`.

This cannot be controlled by setting using named function.

And getting task tree correctly by named functions is tested by `flags-tasks-with-desc.js`.